crates/tx5-connection/src/conn.rs

```diff
@@ -623,7 +624,7 @@ async fn con_task_attempt_webrtc(
                     pub_key = ?task_core.pub_key,
                     a = "webrtc fallback: webrtc processing task closed",
                 );
-                return Fallback(task_core);
+                break;
```

Comment:
The fix. We were deliberately choosing to fall back to the relay when WebRTC fails after being established.

This is what I mean when I complement neonphog's code. You come in looking for an issue and 99% of the code is sensible and you're just looking for a detail to tweak. In this case, the code was expressing exactly what we were seeing happen and not what we want to have happen. Easy to fix.

Reply:
I think this `break;` is in effect equivalent to `return Abort` as is used in the other match arms. So we could potentially use `return Abort` here as well for a bit more consistency?

Reply:
Agree that consistency is good but went the other way and changed all the aborts to break statements. It makes sense to return the fallback because you really want to redirect the control flow but for the termination I think it's a little clearer to let the function abort by finishing.